### ISSUE TITLE
RP2350 does not respond to flash_nuke.uf2

### ISSUE THREAD

**Issue by TRadigk** [raspberrypi/pico-examples#544]
Experiment: Copy over the flash_nuke.uf2 to RP2350 based PICO 2 board 
Observation: nothing happens. 
Expected behaviour: Controller is rebooted and flash is empty. 
Actual behaviour: Nothing happens, the file explorer remains open, until any other uf2 is copied which contains executable code.

**Comment by lurch** [comment]
Seems to work okay for me. Are you sure that you're using the latest https://datasheets.raspberrypi.com/soft/flash_nuke.uf2 (which is now a "[universal binary](https://github.com/raspberrypi/pico-examples?tab=readme-ov-file#universal)") and not the older version (which was RP2040-only) ? Old version had a size of 25600 bytes, new version has a size of 98304 bytes.

**Comment by peterharperuk** [comment]
If you built it for RP2040 it would behave exactly as descrived

**Comment by TRadigk** [comment]
You are right, this page is not updated: https://www.elektronik-kompendium.de/sites/raspberry-pi/2611111.htm
(second google result and first where I learned that this file exists in the first place)

**Comment by lurch** [comment]
You'll be pleased to hear that https://www.raspberrypi.com/documentation/microcontrollers/pico-series.html definitely *is* kept up to date! :joy:

**Comment by TRadigk** [comment]
You got me a new bookmark 😁 👍
